app/javascript/components/Results/ResultGrid/ResultGrid.tsx

```diff
@@ -46,7 +47,19 @@ const getFileType = (fileType?: string) => {
   return (<span className='filetype-label'>{fileType}</span>);
 };
 
-export const ResultGrid = ({ result, affiliate, query, position, vertical }: ResultProps) => {  
+{/* Filter tags UI - to be updated with integration once backend starts sending the data */}
+const getFilterTags = () => {
```

Comment:
@stevenbarragan added this function with a fixed array of tags for now - to be updated once backend is ready to read the dynamic data.

Reply:
Tags come as an array along with each result.

Please update the `SearchResultsLayoutProps` to represent that and pull the tags from here.

```
interface SearchResultsLayoutProps {
  resultsData?: {
    ....
    unboundedResults: boolean;
    results: {
      title: string;
      ...
      tags: string[];
      ...
```

Reply:
code has been updated to handle tags to come from result props